File: 1975/Solution.go
func maxMatrixSum(matrix [][]int) int64 {
    mn, neg, sum := math.MaxInt, 0, 0
    for _, r := range matrix {
        for _, i := range r {
            if i < 0 {
                i *= -1
                neg ^= 1
            }
            sum += i

            mn = min(mn, i)
        }
    }

    return int64(sum - neg * mn * 2)
}

File: 1975/Solution.java
class Solution {
    public long maxMatrixSum(int[][] matrix) {
        long sum = 0;
        int neg = 0, mn = Integer.MAX_VALUE;
        for(int[] r : matrix){
            for(int i : r) {
                if( i < 0) {
                    i *= -1;
                    neg ^= 1;
                }
                sum += i;
                mn = Math.min(mn, i);
            }
        }
        return sum - 2 * neg * mn;
    }
}
